symfony/monolog-bundle/3.1/config/packages/dev/monolog.yaml

```diff
@@ -0,0 +1,19 @@
+monolog:
+    handlers:
+        main:
+            type: stream
+            path: "%kernel.logs_dir%/%kernel.environment%.log"
+            level: debug
+            channels: ["!event"]
+        ## uncomment to get logging in your browser
+        ## you may have to allow bigger header sizes in your Web server configuration
```

Comment:
No need for double `#` here

Reply:
@Pierstoval keep or not the space as well?

Reply:
Simple: someone that uncomments this code may also remove it, so make it like it's a normal comment